INSTALLING.md

```diff
@@ -61,7 +67,7 @@ sudo -E python3 setup_hugepages.py enable && sudo -E python3 setup_hugepages.py
 1. Install git and git-lfs
 
 ```sh
-sudo apt install git git-lfs
+sudo apt install git git-lfs python3-pip
 ```
 
 3. Clone the repo.
```

Comment:
@ttmchiou I believe these steps were different, they also look out of order now (1., 3., 2.). Clone the repo should have instructions on how to clone. Actually looking at Step 3 of Docker, that's how this should look like (with environment variables too). Is it possible there was some typo reorganizing the code?

Reply:
@adifrancescoTT
Whoops, i accidently added python3-pip as a requirement in the Native Host Installation section.
Other than that though, I haven't made any changes to that section so I'm not sure what you mean by instructions on how to clone?

Reply:
moved changes to INSTALLING.md to another ticket.
https://github.com/tenstorrent/tt-metal/issues/8550